kie-wb-smoke-tests/src/test/java/org/kie/smoke/wb/selenium/model/LoginPage.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2015 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.smoke.wb.selenium.model;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPage extends PageObject {
+
+    @FindBy(name = "j_username")
+    private WebElement usernameInput;
+    @FindBy(name = "j_password")
+    private WebElement passwordInput;
+    @FindBy(className = "button")
+    private WebElement loginButton;
+    public final static String BASE_URL = "http://localhost:8080/kie-wb";
```

Comment:
The base URL is being passed to the tests via property (it is currently different for kie-wb and kie-drools-wb, we my need to change that in future), see https://github.com/droolsjbpm/kie-wb-distributions/blob/master/kie-wb-smoke-tests/src/test/java/org/kie/smoke/wb/AbstractWorkbenchIntegrationTest.java#L41 .